src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -24,6 +24,7 @@
 import time
 import uuid
 import webbrowser
+from distutils.version import StrictVersion
```

Comment:
is the package in all platform already? @fengzhou-msft

Reply:
yes, it's part of python libs.